developer-docs/latest/concepts/sso.md

```diff
@@ -0,0 +1,320 @@
+# Single Sign On
```

Comment:
@meganelacheny following our discussion, probably have to add a more visual item to inform of the enterprise edition gold.

And we can't just say `ee` cause it's only in the gold.

I used a bronze one and lost time by not understanding why the feature was not there 😅

Reply:
We were already pointing the need for a gold license there: https://github.com/strapi/documentation/pull/68/files#diff-d3e17514d1e55a57320d0ea2e7e4dad823292a495519a40749c11755742be568R8
But to avoid any confusion I've added a badge there: https://github.com/strapi/documentation/pull/68/files#diff-d3e17514d1e55a57320d0ea2e7e4dad823292a495519a40749c11755742be568R1